src/ott/tools/sliced.py

```diff
@@ -0,0 +1,90 @@
+# Copyright OTT-JAX
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Optional
+
+import jax
+import jax.numpy as jnp
+
+from ott import utils
+from ott.geometry import costs, pointcloud
+from ott.solvers import linear
+
+__all__ = ["sliced_wasserstein"]
+
+Projector_t = Callable[[int, int], jnp.ndarray]
```

Comment:
I'd refactor the projector to `Projector = Callable[[jnp.ndarray], jnp.ndarray]`.